content/source/docs/enterprise/users-teams-organizations/teams.html.md

```diff
@@ -87,4 +84,19 @@ A team can be given read, write, or admin permissions on one or more workspaces.
 - Use any workspace's "Access" tab to manage team permissions on that workspace. For full instructions, see [Managing Access to Workspaces](../workspaces/access.html).
 - For detailed information about the available permissions levels, see [Permissions](./permissions.html).
 
-When determining whether a user can take an action on a resource, TFE uses the highest permission level from that user's teams. (For example, if a user belongs to a team with read permissions on a workspace and another team with admin permissions on that workspace, that user has admin permissions.)
+When determining whether a user can take an action on a resource, TFE uses the highest permission level from that user's teams. For example, if a user belongs to a team with read permissions on a workspace and another team with admin permissions on that workspace, that user has admin permissions.
+
+Organization-level permissions (see [Managing Organization Access](./teams.html#managing-organization-access), below) can also supersede lower workspace permissions. For example, if a user belongs to a team with read permissions on a workspace but also has workspace management enabled, that user has admin permissions on the workspace. Conversely, if the team is allowed to manage the organization's Sentinel policies (which gives read access to all workspaces for enforcing policy sets) and has admin access on the workspace, the higher admin permission level is granted to the workspace.
+
+## Managing Organization Access
+
+-> **API:** See the [Teams API](../api/teams.html). <br/>
+
+A team can be granted permissions to manage Sentinel policies, workspaces, and/or VCS settings across an organization. You can find available options on the team settings page under "Organization Access":
+
+- **Manage Policies**  
+Allows members to create, edit, and delete the organization's Sentinel policies and override soft-mandatory policy checks. Note that this setting implicitly gives read access to all workspaces to set enforcement of [policy sets](../sentinel/manage-policies.html).
+- **Manage Workspaces**  
+Allows members to create and administrate all workspaces within the organization. This is synonymous to giving the [admin permission level](./permissions.html) to all workspaces.
```

Comment:
```suggestion
Allows members to create and administrate all workspaces within the organization. This is synonymous to giving the team [admin permission level](./permissions.html) to all workspaces, plus the ability to create new workspaces (otherwise only available to the owners team). 
```